polyply/src/minimizer.py

```diff
@@ -154,11 +172,26 @@ def target_function(positions):
         return energy
 
     opt_results = scipy.optimize.minimize(target_function, positions, method='L-BFGS-B',
-                                          options={'ftol':0.001, 'maxiter': 100})
+                                          options={'ftol': 0.001, 'maxiter': 100})
 
+    # evaluate if optimization was successful
     positions = opt_results['x'].reshape((-1, 3))
     positions = renew_vs(positions, block, atom_to_idx)
+
     for node_key, idx in atom_to_idx.items():
         coords[node_key] = positions[idx]
 
-    return opt_results['success'], coords
+    for inter_type in inter_types:
+        interactions = block.interactions.get(inter_type, [])
+        for interaction in interactions:
+            atoms = interaction.atoms
+            params = interaction.parameters
+            atom_coords = [positions[atom_to_idx[name]]
+                           for name in atoms]
+            penalty = INTER_METHODS[inter_type](params, atom_coords)
+            if inter_type in ["bonds", "constraints"] and penalty > 10000*tolarance[inter_type]**2.:
```

Comment:
as a minor thing, I think you still have the "tolarance" typo throughout `minimizer.py`

Reply:
damn, I thought I had removed that. Good catch!